src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.stories.tsx

```diff
@@ -30,32 +30,28 @@ const instanceInfo = {
 export const StandardInstanceCreated = Template.bind({});
 StandardInstanceCreated.args = {
   instanceType: "standard",
-  size: 1,
+  size: "x1",
```

Comment:
```suggestion
  size: "1",
```

We should not hint that size matches the streaming unit id, so let's use a generic numeric "name" for it.